tdrs-frontend/src/actions/stts.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+export const FETCH_STTS = 'FETCH_STTS'
+export const SET_STTS = 'SET_STTS'
+export const SET_STTS_ERROR = 'SET_STTS_ERROR'
+export const CLEAR_STTS = 'CLEAR_STTS'
+
+export const fetchStts = () => async (dispatch) => {
+  dispatch({ type: FETCH_STTS })
+  try {
+    const URL = `${process.env.REACT_APP_BACKEND_URL}/stts/alpha`
+    const { data } = await axios.get(URL, {
+      withCredentials: true,
+    })
+    if (data) {
+      dispatch({ type: SET_STTS, payload: { data } })
+    } else {
+      dispatch({ type: CLEAR_STTS })
+    }
+  } catch (error) {
+    dispatch({ type: SET_STTS_ERROR, payload: { error } })
+  }
+}
```

Comment:
This is the stts action that is dispatched from `useEffect` in EditProfile to get the STT's. It gets the alphabetized list and has error handling in case something goes wrong.